File: src/main/java/no/nav/bidrag/aktoerregister/dto/DodsboDTO.java
package no.nav.bidrag.aktoerregister.dto;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
@Schema(description = "Representerer navn for en bidragsaktør.")
public class DodsboDTO {

  @Schema(description = "Navn på kontaktperson for dødsboet.")
  private String kontaktpersion;

  private AdresseDTO adresse;
}

File: src/main/kotlin/no/nav/bidrag/aktoerregister/dto/aktoerregister/dto/DodsboDTO.kt
package no.nav.bidrag.aktoerregister.dto.aktoerregister.dto

import io.swagger.v3.oas.annotations.media.Schema

@Schema(description = "Representerer navn for en bidragsaktør.")
data class DodsboDTO(

    @Schema(description = "Navn på kontaktperson for dødsboet.")
    val kontaktpersion: String? = null,

    val adresse: AdresseDTO? = null
)
